content/en/Platform Deep Dive/Scans/reduced-scope.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Partial Scans: Reduced Scope"
+linkTitle: "Partial Scans: Reduced Scope"
+weight: 50
```

Comment:
The integrations page has the same weight. Now this new page will be under it. Is that the desired order?

Reply:
@mahmoudadly-cobalt i forgot to mention the ordering of the child pages - i think it should definitely be above Integrations (though not sure exactly where - maybe below Sequence Recorder or after Extra Hosts?) but i wasn't sure how to reorder it

Reply:
I updated the order now.